config/locales/mailers/en.yml

```diff
@@ -26,19 +26,22 @@ en:
             part1: "If you've received this message in error, please contact Support at %{support_url}."
             part2: "The Archive of Our Own is a fan-run and fan-supported archive that relies on your donations: %{donate_url}."
         sent_at: "Sent at %{sent_at}."
-    kudos:
+  kudo_mailer:
```

Comment:
These keys are renamed, maybe we need to deal with https://otwarchive.atlassian.net/browse/AO3-6240 first.

Otherwise this looks good.